resources/north-america/canada/MappingWR.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "MappingWR",
+  "type": "slack",
+  "includeLocations": ["waterloo_region.geojson"],
```

Thanks!  Because this area is so small, it is triggering a build warning.. Can we instead get rid of the geojson file and do this instead?  (I'm trying to reduce the size of this library by avoiding bundling a lot of little geojson files).

```suggestion
  "includeLocations": [[-80.5174, 43.4768]],
```

---

done (with a different point)